### ISSUE TITLE
Improve URL Creation

### ISSUE THREAD

**Issue by qalldredge** [gcentral/Website#3]
Improve the handling of unique URLs for each Package and Version.  

This is used to give that URL to someone to have him landing on the right page immediately and for Search Engines indexing since a unique URL will exist for each package version, without having a real HTML web page. This makes the whole code more complex than what it should be.

**Comment by matthiasbaudot** [comment]
Definitely, the code is more complex than it should be just to handle unique URLs for Packages and Versions. This was good for a proof of concept but not good anymore for maintaining something that is going to evolve through collaboration.
I thought about this one and I believe the best way to address this feature is to use JavaScript to parse the URL parameters from the WebVI directly. So we will continue having a single WebVI for all Packages and all Versions, but this WebVI will dynamically load the right content based on the URL parameters.
It might be worth checking this: https://github.com/rajsite/webvi-experiments/tree/master/UrlParameters
I didn't check this code at all but definitely Milan Raj at NI might be our best resource to help with this since I don't have any particular knowledge with JavaScript, and he does.

**Comment by matthiasbaudot** [comment]
I got a reply from NI and they made available some code for us to help with this issue.
This code is originally available at: https://github.com/rajsite/webvi-experiments/tree/master/URL

I've added this piece of code to the GCentral project and refactored the existing Web Application and Web Services to simplify the whole Package Unique URLs creation.

Now, the PackageDetails WebVI is able by itself to parse the URL search parameters so it knows which Package and Version to load in its UI.

Changes have been merged to master and the new website implementation has been pushed live.
